app/src/main/java/com/xmartlabs/scasas/doapp/controller/TaskController.java

```diff
@@ -45,4 +53,70 @@
         .observeOn(AndroidSchedulers.mainThread())
         .subscribeOn(Schedulers.io());
   }
+
+  public Single<Long> getTasksCount(User user, Group group) {
+    return Single
+        .fromCallable(() -> SQLite.selectCountOf()
+            .from(Task.class)
+            .where(Task_Table.user_id.eq(user.getId()))
+            .and(Task_Table.group_id.eq(group.getId()))
+            .count()
+        )
+        .observeOn(AndroidSchedulers.mainThread())
+        .subscribeOn(Schedulers.io());
+  }
+
+  public Single<Long> getTasksCountForASpecificMonth(User user, LocalDate date) {
+    return Single
+        .fromCallable(() -> SQLite.selectCountOf()
+            .from(Task.class)
+            .where(Task_Table.user_id.eq(user.getId()))
+            .and(Task_Table.date.greaterThanOrEq(date))
+            .and(Task_Table.date.lessThan(date.plusMonths(1)))
+            .count()
+        )
+        .observeOn(AndroidSchedulers.mainThread())
+        .subscribeOn(Schedulers.io());
+  }
+
+  public Single<Long> getFinishedTasksCount(User user, Group group) {
+    return Single
+        .fromCallable(() -> SQLite.selectCountOf()
+            .from(Task.class)
+            .where(Task_Table.user_id.eq(user.getId()))
+            .and(Task_Table.group_id.eq(group.getId()))
+            .and(Task_Table.isFinished.eq(true))
+            .count()
+        )
+        .observeOn(AndroidSchedulers.mainThread())
+        .subscribeOn(Schedulers.io());
+  }
+
+  public Single<Long> getFinishedTasksCountForASpecificMonth(User user, LocalDate date) {
+    return Single
+        .fromCallable(() -> SQLite.selectCountOf()
+            .from(Task.class)
+            .where(Task_Table.user_id.eq(user.getId()))
+            .and(Task_Table.date.greaterThanOrEq(date))
+            .and(Task_Table.date.lessThan(date.plusMonths(1)))
+            .and(Task_Table.isFinished.eq(true))
+            .count()
+        )
+        .observeOn(AndroidSchedulers.mainThread())
+        .subscribeOn(Schedulers.io());
+  }
+
+  public Single<Long> getIntFinishPercentage(User user, LocalDate date) {
```

Comment:
Please rename. It's not returning int.
Also, you should avoid as much as possible adding type information in names. It should be clear in the type itself